### PR TITLE
[CH2-41] iframe size

### PR DESCRIPTION
**Pull request by tim-yao**
**JIRA issue:** https://digital-engagement.atlassian.net/browse/CH2-41

### Changed

1.  Respect the CMS height settings for iframe
2. Changed default iframe in markup from fixed ratio to reconfigured height, so mobile has better usabilility

### CODE REVIEW

File: packages/components/Organisms/Markup/Markup.vue [hunk 1]
```diff
@@ -125,11 +125,27 @@ $responsive-iframe-padding-top: $rpl-space-4 !default;
       @include rpl_text_color($rpl-markup-link-color);
     }
   }
+
   /* Iframes */
   &__iframe-container {
-    @include rpl_responsive_iframe;
```
Comment (rubyllamadora) [inline packages/components/Organisms/Markup/Markup.vue:130]: safe to delete the scss file if no longer used?

File: packages/ripple-nuxt-tide/lib/config/markup-plugins.js [hunk 2]
```diff
@@ -116,6 +116,10 @@ const pluginIframe = function () {
   const wrapperClasses = ['rpl-markup__iframe-container']
   this.find('iframe').map((i, el) => {
     const iframe = this.find(el)
+    // If no height setting from CMS, we give it a default height.
+    if (!iframe.attr('height')) {
+      wrapperClasses.push('rpl-markup__iframe-container--default')
```
Comment (rubyllamadora) [inline packages/ripple-nuxt-tide/lib/config/markup-plugins.js:121]: 👍

**Review by rubyllamadora — APPROVED** [review]
Looks fine to me @tim-yao (:

File: packages/components/Organisms/Markup/Markup.vue (hunk 1 above)
Comment (tim-yao) [reply packages/components/Organisms/Markup/Markup.vue:130]: > safe to delete the scss file if no longer used?

Thanks @rubyllamadora , redundant css now is removed.

**Pull request merged**